include/nbl/system/CSystemLinux.h

```diff
@@ -16,6 +16,9 @@ class CSystemLinux final : public ISystemPOSIX
 
 		NBL_API2 SystemInfo getSystemInfo() const override;
 };
+
+bool isDebuggerAttached();
```

Comment:
NBL_API2

Reply:
Also why is this a random global function? We don't do those.

add it as `static` method to `ISystem` and implement in `ISystem.cpp`